DQM/CTPPS/plugins/TotemT2DQMSource.cc

```diff
@@ -58,7 +58,8 @@ class TotemT2DQMSource : public DQMEDAnalyzer {
   static constexpr double T2_BIN_WIDTH_NS_ = 25. / 4;
   MonitorElement* totemT2ErrorFlags_2D_ = nullptr;
 
-  enum evFlag { t2TE = 0, t2LE, t2MT, t2ML };
+  typedef int evFlag;
```

Comment:
This type name seems be unused, so probably could be removed.

Reply:
OK.